internal/pkg/agent/application/upgrade/step_download.go

```diff
@@ -127,7 +122,7 @@ func (u *Upgrader) downloadArtifact(ctx context.Context, version, sourceURI stri
 	return path, nil
 }
 
-func (u *Upgrader) appendFallbackPGP(targetVersion string, pgpBytes []string) []string {
+func (u *Upgrader) appendFallbackPGP(tpv *agtversion.ParsedSemVer, pgpBytes []string) []string {
```

Comment:
Could we just continue to call it `targetVersion` instead of the less descriptive `tpv` please? 🙂

Reply:
b852a8371293bfcd808ddc63e0e1488e6c11bb09